src/search.cpp

```diff
@@ -358,13 +348,15 @@ void Thread::search() {
 
   multiPV = std::min(multiPV, rootMoves.size());
 
+  int h = (idx - 1) % 20; // helper index, cycle after 20 threads
```

Comment:
Naming variables is a matter of taste, so I will let maintainers decide which names are the best.